portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,25 +14,46 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Query;
```

Comment:
I don't think this class exists. And I think you meant to add 
`import com.google.appengine.api.datastore.Query;` to the `DeleteServlet`.

Reply:
Ah I've been gettting lazy whoops, I really need to compile before committing.